react-native-libraries.json

```diff
@@ -2413,7 +2413,7 @@
     ],
     "ios": true,
     "android": true,
-    "expo": true
+    "expo": false
```

Comment:
```suggestion
```

We can just removed the falsy property.

Reply:
Thanks! I didn't look at other objects as a basis for conformity and will do so in the future.